azure/defaults.go

```diff
@@ -97,6 +98,16 @@ func GenerateBackendAddressPoolName(lbName string) string {
 	return fmt.Sprintf("%s-%s", lbName, "backendPool")
 }
 
+// GenerateScope generates the scope name.
```

Comment:
```suggestion
// GenerateSubscriptionScope generates a role assignment scope that applies to all resources in the subscription.
```
to make it more descriptive on what kind of scope it's generating (especially since the word "scope" is overused in this codebase)